src/brevitas/graph/gpxq.py

```diff
@@ -98,7 +98,7 @@ def __enter__(self):
 
                 # Attach hooks for GPTQ
                 if self._is_module_supported(module):
-                    gpxq = self.class_implementation(
+                    gpxq = self.init_class(
```

Comment:
let's call the function `initialize_module_optimizer`
And then this line would be:
gpxq_module_optimizer = self.initialize_module_optimizer